media/css/hubs/_cards.scss

```diff
@@ -0,0 +1,309 @@
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+
+@import 'cta-link';
+
+$color-card-cta: #9C432A;
+$card-padding: 20px;
+
+.card {
+    padding: 20px 0 40px;
+
+    &.card-stacked {
+        border-top: 1px solid #000;
+    }
+}
+
+.card-heading {
+    text-transform: lowercase;
+}
+
+// the copy & cta for a card (not the .card-label)
+.card-content {
+    background: #fff;
+    padding: 20px 20px 20px $card-padding;
+
+    @media #{$mq-tablet} {
+        padding-left: $card-padding * 2;
+    }
+}
+
+// the little highlighted bit of text at the top of a card
+.card-label {
+    @include font-size(24px);
+    background: #000;
+    color: #fff;
+    display: inline-block;
+    line-height: 1.1;
+    margin-bottom: 20px;
+    padding: 0 5px 0 $card-padding;
+    text-transform: lowercase;
+
+    @media #{$mq-tablet} {
+        padding-left: $card-padding * 3;
+    }
+}
+
+// the primary link within .card-content
+.card-cta {
+    @extend .cta-link;
+}
+
+.featured-image-card {
+    background-position: top left;
+    background-repeat: no-repeat;
+    background-size: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    margin-bottom: 20px;
+    overflow: hidden;
+    padding-bottom: 0;
+    position: relative;
+
+    .card-content,
+    .card-label {
+        position: relative;
+        z-index: 2;
+    }
+
+    .card-label {
+        background: #fff;
+        color: #000;
+        margin-bottom: 20px;
+    }
+
+    .card-content {
+        margin-right: $card-padding;
+        margin-top: 40px;
+        padding-top: 40px;
+    }
+
+    @media #{$mq-phone-wide} {
+        .card-content {
+            margin-top: 80px;
+        }
+    }
+
+    @media #{$mq-tablet} {
+        .card-content {
+            margin-right: $card-padding * 2;
+        }
+    }
+
+    @media #{$mq-desktop} {
+        .card-content {
+            margin-top: 200px;
+        }
+    }
+
+    @supports (display: grid) {
+        @media #{$mq-desktop} {
+            margin-bottom: 0;
+            padding-bottom: 10%;
+
+            .card-content {
+                margin-top: auto;
+            }
+        }
+
+        @media #{$mq-desktop-wide} {
+            padding-bottom: 0;
+        }
+    }
+}
+
+// specific styles for a card with an image. used with .card
+.image-card {
+    .card-content-wrapper {
+        background-position: top left;
+        background-repeat: no-repeat;
+        padding-top: 120px;
+    }
+
+    .card-content {
+        margin-right: $card-padding;
+    }
+
+    @media #{$mq-tablet} {
+        .card-content {
+            margin-right: auto;
+            padding-bottom: $card-padding * 3;
+            padding-right: $card-padding * 2;
+            padding-top: 60px;
+            width: 40%;
+        }
+    }
+}
+
+.three-card-monte {
```

Comment:
I think we should converge this code, but I'd rather not dive into that now. We can do this during the refactor/code cleanup next sprint?